content/concepts/server-api.md

```diff
@@ -550,6 +551,32 @@ $ curl \
 }
 ```
 
+### HMAC
+
+
+| Method   | Path                  | Content-Type       |
+|:--------:|:---------------------:|:------------------:|
+| `PUT`    | `/v1/key/hmac/`       | `application/json` |
+
+Compute a message authentication code (MAC) for some provided data. 
```

Comment:
Is there anything more specific we can say about this `some` data? Perhaps someone from eng has thoughts.

Reply:
I think it might be literally for the provided data blob?

```suggestion
Compute a message authentication code (MAC) for the data provided in the POST data.
```

Reply:
Yes, what Ravind suggested.
The request passes the data string to compute the HMAC for.

So it's literally whatever data you pass in the request.

Reply:
Rephrased the description to clarify that it is the data passed in the request.